### PR TITLE
[May18] Cursor light fix (and other minor fixes)

### PR DESCRIPTION
**Pull request by keveleigh**
Overview
---
Some minor updates that were found while updating the Academy courses to the most recent Toolkit release.

Partner PR to #2036.

Changes
---
- Fixes: #1983.

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Scripts/Billboard.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@ public class Billboard : MonoBehaviour
         /// The axis about which the object will rotate.
         /// </summary>
         [Tooltip("Specifies the axis about which the object will rotate.")]
+        [SerializeField]
         private PivotAxis pivotAxis = PivotAxis.XY;
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: See Comment from: https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2036#discussion_r185647500

Comment (keveleigh) [reply Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: This one is private.

Comment (StephenHodgson) [reply Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: Then the other should be too.

Comment (keveleigh) [reply Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: Agreed!

Comment (StephenHodgson) [reply Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: I saw you did it already 😉

**Pull request merged**